### PR TITLE
Fix filtering problems by full AC

### PR DESCRIPTION
**Pull request by int-y1**
* Give a green icon to >100% AC subs
* Fix filtering solved problems

### CODE REVIEW

File: judge/utils/problems.py [hunk 1]
```diff
@@ -24,8 +24,8 @@ def contest_completed_ids(participation):
     key = 'contest_complete:%d' % participation.id
     result = cache.get(key)
     if result is None:
-        result = set(participation.submissions.filter(submission__result='AC', points=F('problem__points'))
-                     .values_list('problem__problem__id', flat=True).distinct())
+        result = set(participation.submissions.filter(submission__result='AC', points__gte=F('problem__points'))
```
Comment (Ninjaclasher) [inline judge/utils/problems.py:27]: Could we perhaps use `case_points` / `case_total` here as well?
```suggestion
        result = set(participation.submissions.filter(submission__result='AC', submission__case_points=F('submission__case_total'))
```

Comment (int-y1) [reply judge/utils/problems.py:27]: This is a ContestSubmission. Is it still ok to check `case_points`/`case_total`?

**Review by Ninjaclasher — COMMENTED** [review]
LGTM

**Review by kiritofeng — APPROVED** [review]
LGTM, thanks!

**Pull request merged**